### PR TITLE
:seedling: Add the unit tests in pkg/model/config/config_test.go for HasWebhook 

### PR DESCRIPTION
**Pull request by prafull01**
**Description**
Add the HasWebhook() unit tests to improve the coverage.

**Motivation**
Follow up of the PR #1903 where the HasWebhook() function is introduced.

### CODE REVIEW

File: pkg/model/config/config_test.go [hunk 1]
```diff
@@ -295,4 +296,17 @@ var _ = Describe("Config", func() {
...
 
 	})
+
+	Context("HasWebhook", func() {
+		It("should return true when config has a webhook with the GVK", func() {
+			c.UpdateResources(gvk3)
+			Expect(c.Resources).To(Equal([]ResourceData{gvk3}))
+			Expect(c.HasWebhook(gvk3)).To(BeTrue())
+		})
+		It("should return false when config has a webhook with the GVK", func() {
```
Comment (camilamacedo86) [inline pkg/model/config/config_test.go:306]:
```suggestion
		It("should return false when config has NOT a webhook with the GVK", func() {
```

Comment (prafull01) [reply pkg/model/config/config_test.go]: Done

File: pkg/model/config/config_test.go [hunk 2]
```diff
@@ -295,4 +296,17 @@ var _ = Describe("Config", func() {
 		})
 
 	})
+
+	Context("HasWebhook", func() {
+		It("should return true when config has a webhook with the GVK", func() {
```
Comment (Adirio) [inline pkg/model/config/config_test.go:301]:
```suggestion
		It("should return true when config has a webhook for the GVK", func() {
```

File: pkg/model/config/config_test.go [hunk 3]
```diff
@@ -295,4 +296,17 @@ var _ = Describe("Config", func() {
...
+			c.UpdateResources(gvk3)
+			Expect(c.Resources).To(Equal([]ResourceData{gvk3}))
+			Expect(c.HasWebhook(gvk3)).To(BeTrue())
+		})
+		It("should return false when config NOT has a webhook with the GVK", func() {
```
Comment (Adirio) [inline pkg/model/config/config_test.go:306]: Any of the following options:
```suggestion
		It("should return false when config does not have a webhook for the GVK", func() {
```
```suggestion
		It("should return false when config has no webhook for the GVK", func() {
```

**Review by Adirio — CHANGES_REQUESTED** [review]
Some english related comments:

**Comment by prafull01** [comment]
> Some english related comments:

@Adirio Fixed those.

**Comment by camilamacedo86** [comment]
Hi @prafull01,

All fine here. Since we are only adding tests the correct mojo is: `Infra/Tests/Other: 🌱 (:seedling:)`.
I will update it for we are able to merge this one.

**Review by camilamacedo86 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**